src/generics/generics.lisp

```diff
@@ -23,3 +22,30 @@ GEB> (gapply (comp
              (right so1))
 (left s-1)
 ```"))
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Conversion functions
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+(defgeneric to-bitc (morphism)
```

Comment:
Not currently of importance, but somewhere down the line probably a good idea to include some more documentation specifying the passes this makes.

Reply:
Would be good, and a documentation string on each instance as well to explain how it can be used.

I may want to open a second PR that is based on this and the lambda changes that removes the trans ordering